src/GraphQL.MemoryCache/MemoryDocumentCacheOptions.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using Microsoft.Extensions.Options;
+
+namespace GraphQL.Caching
+{
+    /// <summary>
+    /// Provides configuration options for <see cref="MemoryDocumentCache"/>.
+    /// </summary>
+    public class MemoryDocumentCacheOptions : IOptions<MemoryDocumentCacheOptions>
+    {
+        /// <summary>
+        /// The maximum total length of all queries cached. Assume maximum memory used is about 10x this value.
+        /// </summary>
+        public int MaxTotalQueryLength { get; set; }
```

Comment:
It is always encouraged to give some meaningful defaults to all options. I suggest `100;` for `MaxTotalQueryLength` and `TimeSpan.FromMinutes(10)` for `SlidingExpiration`. The default configuration is currently broken.

Reply:
Fine, but 100 ? An introspection query is 1.3kb.  The kitchensink query is 800 bytes.  I would have to say that it is not meaningful if set to 100.  Perhaps 100,000?

Reply:
Any value != `null` suits me.